spring-ai-core/src/test/java/org/springframework/ai/converter/BeanOutputConverterTest.java

```diff
@@ -112,6 +116,15 @@ String getSomeString() {
 
 	}
 
+	@JsonPropertyOrder({ "string_property", "foo_property", "bar_property" })
+	record TestClassWithJsonPropertyOrder(
+			@JsonProperty("string_property") @JsonPropertyDescription("string_property_description") String someString,
+
+			@JsonProperty(required = true, value = "foo_property") String foo,
```

Comment:
Just as a validation, we could perhaps change the order here to make sure the actual order from "JsonPropertyOrder" dictates the ordering. will change it when merging.